apps/andi/lib/andi_web/live/edit_live_view.ex

```diff
@@ -127,13 +148,15 @@ defmodule AndiWeb.EditLiveView do
     |> FormTools.adjust_source_query_params_for_url()
     |> InputConverter.form_data_to_ui_changeset()
     |> complete_validation(socket)
+    |> mark_changes()
   end
 
   def handle_event("validate", %{"form_data" => form_data, "_target" => ["form_data", "technical", "sourceQueryParams" | _]}, socket) do
     form_data
     |> FormTools.adjust_source_url_for_query_params()
     |> InputConverter.form_data_to_ui_changeset()
     |> complete_validation(socket)
```

Comment:
Are there any cases of `complete_validation` that are not paired with `mark_changes`. If not, I'd strongly consider just calling it from that function, unless we feel like this is more explicit.

Reply:
It gets called on save as well. That's why i just made it separate

Reply:
That'll do. That'll do